Library/Formula/synergy.rb

```diff
@@ -0,0 +1,63 @@
+class Synergy < Formula
+  homepage "http://synergy-project.org"
+  url "https://github.com/synergy/synergy/archive/1.6.3.tar.gz"
+  sha256 "93b1965b8e0cfc55cc654aab7053a58b9e730d09e68bbc520be32353a01dce1a"
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+
+  patch :DATA
```

Comment:
Well, the patches are still being reviewed, and won't be included before release 1.6.4.

That means we could keep the 1.6.3 formula until it's deprecated by the new version. 

I'm getting notifications of new releases from synergy, so I'll be able to update the formula as soon as it is released.